_scicomputing/store_objectstore.md

```diff
@@ -37,34 +38,52 @@ An additional tool is [Motuz](http://motuz.fredhutch.org), a tool that facilitat
 
-In the event that you need to share a very large dataset (e.g. >5TB) with an external collaborator and the collaborator intends to store that dataset within their own institutions S3 bucket, please contact SciComp for assistance.  There are ways to copy data between S3 buckets far more quickly than would otherwise be possible with any of the methods described here, however these typically require coordination between the collaborator's IT department and the SciComp team.  This can also be done for receiving very large datasets from external S3 buckets as well.  It is very rare that this method of bulk-copying data between buckets is required, however it is available if the situation warrants it.  Typically, the setup for this will take up to 48 hours.  The transfer speeds depend on a number of factors, however this has been reliably and repeatedly measured at 30TB-40TB per hour for well over 1 million files.
+### Managed S3 Buckets
 
-#### How it Works
+Managed S3 buckets in S3 are created and named for each investigator account.  As an example, if you work in the lab of the PI Jane Doe, your lab's S3 bucket name will be `fh-pi-doe-j-eco` for the private bucket and `fh-pi-doe-j-eco-public` for the public bucket. Please substitute your lab's actual bucket name when using the examples in our Resource Library demos.  These managed buckets have a standard configuration and are subidized by the Hutch.
 
-S3 (the Simple Storage Service) is an object store very much like the Economy file service, though provided by Amazon Web Services.  Storage resources in S3 are organized much like the other Fred Hutch Object and Block systems, with a "PI bucket" for each investigator at the Hutch which is analogous to the investigator directories available in the traditional file system. A specialized client (the AWS command line interface) is used to upload the data from local storage to S3 storage.
+A specialized client is required to upload data into S3 storage.
```

Comment:
Might want to link here to cyberduck, mountain duck page as well as the command line clients pages.  They are in the compdemos folder I believe.

Reply:
How does e0bb39e look?